BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -472,7 +472,7 @@ Protocol developers are strongly encouraged to register an ALPN identifier for t
 ## Zero Round Trip Time (0-RTT) Data in TLS 1.3
 
 The 0-RTT early data feature is new in TLS 1.3. It provides reduced latency
-when TLS connections are resumed, at the potential cost of security.
+when TLS connections are resumed, at the potential cost of certain security properties.
```

Comment:
"cost of security" sounds like all security is lost; the reality is much more subtle.